packages/sui-mono/src/conventional-changelog.js

```diff
@@ -0,0 +1,152 @@
+'use strict'
+
+const addStream = require('add-stream')
+const gitRawCommits = require('git-raw-commits')
+const conventionalCommitsParser = require('conventional-commits-parser')
+const conventionalChangelogWriter = require('conventional-changelog-writer')
+const _ = require('lodash')
+const stream = require('stream')
+const through = require('through2')
+const execFileSync = require('child_process').execFileSync
+
+const mergeConfig = require('./merge-config.js')
+function conventionalChangelog(options, context, gitRawCommitsOpts, parserOpts, writerOpts, gitRawExecOpts) {
```

Comment:
We needed to "fork" conventional-changelog-core library for now since there is a bug that doesn't take into consideration custom tags parameter. We need because we have custom tag name for our packages (we don't use lerna standard)